settings/_example.settings.global.scss

```diff
@@ -6,4 +6,4 @@
 
 // Standardise some UI treatments.
 $global-radius: 3px !default;
-$global-transition: (1/3) + s !default;
+$global-transition: 300ms !default;
```

Comment:
I would like to enhance it to like what @fuhlig suggested [here](https://github.com/inuitcss/inuitcss/issues/214#issue-184887167) in his simple example:

```scss
$global-transition: all 0.3s ease;
```

I think when we just set a duration as a value, the variable should be named `$global-transition-duration` instead.